### ISSUE TITLE
No search results if you type in metadata keywords as a search query

### ISSUE THREAD

**Issue by jtamiace** [learningequality/kolibri#6910]
### Observed behavior

![Screenshot_20200519-105616](https://user-images.githubusercontent.com/6668144/82579022-0c9d3e80-9b42-11ea-9145-33735d08d3ca.png)

### Expected behavior

Expected some videos to show

### User-facing consequences

User may not think there are any videos on the device

### Errors and logs

…

### Steps to reproduce

Enter a resource type as a query for search in Learn

### Context

Kolibri 0.13.3
Android app

**Comment by jtamiace** [comment]
I don't think we _want_ to show videos as search results, though. Unclear whether we should handle this kind of scenario

**Comment by rtibbles** [comment]
One suggestion that I have made for this sort of thing previously is for our search to pick up filter terms from the search box and offer the ability to turn them into a filter - GMail search does something similar.

**Comment by indirectlylit** [comment]
this is essentially a duplicate of https://github.com/learningequality/kolibri/issues/4796
